test/test_assets/css.json

```diff
@@ -0,0 +1 @@
+{"dist/bundle.css":"/assets/dist/bundle.css"}
```

Comment:
it'd be nice if manifests had a trailing `\n` (this is true for any text file, because `echo` for example)